### PR TITLE
Changes to the ReadMe.Md file

### PR DESCRIPTION
**Pull request by DerekRoy**
This update focuses on eliminating pronouns such as you and we where possible, deleting words that are ambiguous, and shifting the writing style from passive to active voice. The goal is to make it very easy for users to follow and understand by cutting superfluous or ambiguous words.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,69 +17,69 @@
...
 all the familiar building blocks of the classic Jupyter Notebook (notebook,
 terminal, text editor, file browser, rich outputs, etc.) in a flexible and
 powerful user interface.
-Eventually, JupyterLab will replace the classic Jupyter Notebook.
+JupyterLab will eventually replace the classic Jupyter Notebook.
 
-JupyterLab can be extended using extensions that are [npm](https://www.npmjs.com/) packages
-and use our public APIs. You can search for the GitHub topic [jupyterlab-extension](https://github.com/topics/jupyterlab-extension) to find extensions. To learn more about extensions, see our [user documentation](https://jupyterlab.readthedocs.io/en/latest/user/extensions.html).
+JupyterLab can be extended using [npm](https://www.npmjs.com/) packages
+and our public APIs. To find extensions search for the GitHub topic [jupyterlab-extension](https://github.com/topics/jupyterlab-extension). To learn more about extensions, see the [user documentation](https://jupyterlab.readthedocs.io/en/latest/user/extensions.html).
```
Comment (jasongrout) [inline README.md]:
```suggestion
that use our public APIs. To find JupyterLab extensions, search for the npm keyword [jupyterlab-extension](https://www.npmjs.com/search?q=keywords:jupyterlab-extension) or the GitHub topic [jupyterlab-extension](https://github.com/topics/jupyterlab-extension). To learn more about extensions, see the [user documentation](https://jupyterlab.readthedocs.io/en/latest/user/extensions.html).
```

File: README.md [hunk 2]
```diff
@@ -17,69 +17,69 @@
...
 
 ## Getting started
 
 ### Installation
 
-You can [install](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) JupyterLab using `conda`, `pip`, or `pipenv`. We recommend using conda if you do not have a preference.
+[install](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) JupyterLab using `conda`, `pip`, or `pipenv`. Conda is recommended if you have no installation preference.
 
-Instructions on how to install the project from the git sources are available in our [contributor documentation](CONTRIBUTING.md).
+Project instalation instructions from the git sources are available in the [contributor documentation](CONTRIBUTING.md).
```
Comment (jasongrout) [inline README.md]:
```suggestion
Project installation instructions from the git sources are available in the [contributor documentation](CONTRIBUTING.md).
```

File: README.md [hunk 3]
```diff
@@ -17,69 +17,69 @@
...
 
 #### conda
 
-Conda is an open source package management system and environment management system that runs on Windows, macOS and Linux. It can package and distribute software for any language, and by default uses the Anaconda repository managed by Anaconda, Inc. If you wish to use conda and do not have it, [see the conda installation instructions](https://conda.io/docs/user-guide/install/index.html).
+Conda is an open source package management system and environment management system that runs on Windows, macOS, and Linux. Conda packages and distributes software for any language, and by default uses the Anaconda repository managed by Anaconda Inc. To install conda please [see the conda installation instructions](https://conda.io/docs/user-guide/install/index.html).
```
Comment (jasongrout) [inline README.md]:
```suggestion
Conda is an open source package management system and environment management system that runs on Windows, macOS, and Linux. Conda packages and distributes software for any language, and by default uses the Anaconda repository managed by Anaconda Inc. To install conda, please [see the conda installation instructions](https://conda.io/docs/user-guide/install/index.html).
```

File: README.md [hunk 4]
```diff
@@ -17,69 +17,69 @@
...
 
-If you use [`conda`](https://anaconda.org/conda-forge/jupyterlab), you can install as:
+Use [`conda`](https://anaconda.org/conda-forge/jupyterlab), and install with the command:
```
Comment (jasongrout) [inline README.md]:
```suggestion
Install the [JupyterLab `conda` package](https://anaconda.org/conda-forge/jupyterlab) with:
```

File: README.md [hunk 5]
```diff
@@ -17,69 +17,69 @@
...
+Adding `--user` after `pip install` will install the files to a local user install directory (typically `~/.local/` or `%APPDATA%\Python` on Windows) instead of the system-wide directory. This can be helpful, especially if writing to the system-wide directory is not permitted. However, the user-level `bin` directory must be added to the `PATH` environment variable in order to launch `jupyter lab`.
 
 #### pipenv
 
-`Pipenv` is intended to provide users and developers of applications with an easy method to setup a working environment. You must have Python installed first. See the [pipenv installation documentation](https://docs.pipenv.org/install) if you wish to use it but do not have it installed.
+`Pipenv` provides users and developers of applications with an easy method to setup a working environment, however Python must be installed first. See the [pipenv installation documentation](https://docs.pipenv.org/install) to use Pipenv if it is not installed.
 
-If you use `pipenv`, you can install it as:
+`pipenv`, can be installed as:
```
Comment (jasongrout) [inline README.md]:
```suggestion
`pipenv` can be installed as:
```

File: README.md [hunk 6]
```diff
@@ -117,11 +117,11 @@ Start up JupyterLab using:
 jupyter lab
 ```
 
-JupyterLab will open automatically in your browser. See our [documentation](http://jupyterlab.readthedocs.io/en/stable/getting_started/starting.html) for additional details.
+JupyterLab will open automatically in the browser. See the [documentation](http://jupyterlab.readthedocs.io/en/stable/getting_started/starting.html) for additional details.
 
 ### Prerequisites and Supported Browsers
 
-Jupyter notebook version 4.3 or later. To check the notebook version:
+Jupyter notebook version 4.3 or later is required. To check the notebook version run the command:
```
Comment (jasongrout) [inline README.md]:
```suggestion
Jupyter notebook version 4.3 or later is required. To check the notebook version, run the command:
```

File: README.md [hunk 7]
```diff
@@ -141,22 +141,22 @@ See our [documentation](http://jupyterlab.readthedocs.io/en/latest/getting_start
...
 
-If you would like to contribute to the project, please read our [contributor documentation](CONTRIBUTING.md).
+To contribute to the project, please read the [contributor documentation](CONTRIBUTING.md).
 
 JupyterLab follows the Jupyter [Community Guides](https://jupyter.readthedocs.io/en/latest/community/content-community.html).
 
 ### Extending JupyterLab
 
-To start developing your own extension, see our [developers documentation](https://jupyterlab.readthedocs.io/en/latest/developer/extension_dev.html) and [API docs](http://jupyterlab.github.io/jupyterlab/index.html).
+To start developing extension, see the [developers documentation](https://jupyterlab.readthedocs.io/en/latest/developer/extension_dev.html) and [API docs](http://jupyterlab.github.io/jupyterlab/index.html).
```
Comment (jasongrout) [inline README.md]:
```suggestion
To start developing an extension, see the [developer documentation](https://jupyterlab.readthedocs.io/en/latest/developer/extension_dev.html) and the [API docs](http://jupyterlab.github.io/jupyterlab/index.html).
```

File: README.md [hunk 8]
```diff
@@ -141,22 +141,22 @@ See our [documentation](http://jupyterlab.readthedocs.io/en/latest/getting_start
...
 
 ### License
 
-We use a shared copyright model that enables all contributors to maintain the
+A shared copyright model enables all contributors to maintain the
```
Comment (jasongrout) [inline README.md]:
```suggestion
JupyterLab uses a shared copyright model that enables all contributors to maintain the
```

**Comment by jasongrout** [comment]
This looks great! Thanks so much! I added a few more edits for your consideration (trying out the new GitHub patch feature). I see you already merged them. I'll wait for tests to pass before merging.

**Comment by jasongrout** [comment]
Looks like the linter is complaining (probably about whitespace or something?) Can you do what it suggests at https://travis-ci.org/jupyterlab/jupyterlab/jobs/453127631#L677: run `jlpm run lint` and commit the changes to the readme?

**Comment by blink1073** [comment]
Closing in favor of #5701 that adds a linting commit (maintainers can't push to a master branch).  Thanks again @DerekRoy!

**Pull request closed without merging**